src/ol/reproj/reproj.js

```diff
@@ -18,8 +20,9 @@ goog.require('ol.proj');
  * @type {boolean}
  * @private
  */
-ol.reproj.browserAntialiasesClip_ = !goog.labs.userAgent.browser.isChrome() ||
-                                    goog.labs.userAgent.platform.isIos();
+ol.reproj.browserAntialiasesClip_ = !(!winNav.userAgent.match('CriOS') && window.chrome !== null &&
+    window.chrome !== undefined && winNav.vendor === 'Google Inc.' &&
+    isOpera == false && isIEedge == false);
```

Comment:
You could use an immediately invoked function expression here.  So, something like:

```js
ol.reproj. browserAntialiasesClip_ = (function() {
  var winNav = window.navigator;
  var isOpera = winNav.userAgent.indexOf('OPR') > -1;
  var isIEedge = winNav.userAgent.indexOf('Edge') > -1;
  return // ...
}
```